lib/src/screens/wallet_list/wallet_list_page.dart

```diff
@@ -241,17 +242,36 @@ class WalletListBodyState extends State<WalletListBody> {
             .wallet_list_failed_to_load(wallet.name, e.toString()));
       }
     });
+    }else{
+      try {
+        changeProcessText(S.of(context).wallet_list_loading_wallet(wallet.name));
+        await widget.walletListViewModel.loadWallet(wallet);
+        hideProgressText();
+        Navigator.of(context).pop(); 
+      } catch (e) {
+        changeProcessText(S
+            .of(context)
+            .wallet_list_failed_to_load(wallet.name, e.toString()));  
+      }
+    }
   }
 
   Future<void> _removeWallet(WalletListItem wallet) async {
-    await Navigator.of(context).pushNamed(Routes.auth, arguments:
+    if(widget.walletListViewModel.checkIfAuthRequired()){
+     await Navigator.of(context).pushNamed(Routes.auth, arguments:
         (bool isAuthenticatedSuccessfully, AuthPageState auth) async {
       if (!isAuthenticatedSuccessfully) {
         return;
       }
+      _onSuccessfulAuth(wallet, auth);
+    });
+  }else{
+      _onSuccessfulAuth(wallet, null);
+    }
+  }
```

Comment:
Why here is so strange left offsets ? Add spaces between of parts of the operators. Why `}else{` is on same range as ending closing function brace ?